instaclustr/resource_cluster.go

```diff
@@ -145,10 +172,12 @@ func resourceCluster() *schema.Resource {
 						"bundle": {
 							Type:     schema.TypeString,
 							Required: true,
+							ForceNew: true,
 						},
 						"version": {
 							Type:     schema.TypeString,
 							Required: true,
+							ForceNew: true,
 						},
 						"options": {
 							Type:     schema.TypeMap,
```

Comment:
Is there any reason we don't include `ForceNew` on this composite object? Same question applies to others of similar situation.